src/java.desktop/share/classes/javax/swing/plaf/synth/SynthDefaultLookup.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2003, 2025, Oracle and/or its affiliates. All rights reserved.
```
It's the third day of 2025.